crates/fluvio-dataplane-protocol/src/error_code.rs

```diff
@@ -35,6 +35,9 @@ pub enum ErrorCode {
     #[fluvio(tag = 6)]
     #[error("the given SPU is not the leader for the partition")]
     NotLeaderForPartition,
+    #[fluvio(tag = 7)]
+    #[error("the request timed out.")]
+    RequestTimedOut,
```

Comment:
Can we add timeout period and what kind?

Reply:
Added